src/NuGet.Core/NuGet.Packaging/Signing/Authoring/SignPackageRequest.cs

```diff
@@ -27,5 +27,12 @@ public class SignPackageRequest
         /// </summary>
         public X509Certificate2 Certificate { get; set; }
 
+#if IS_DESKTOP
+        /// <summary>
+        /// PrivateKey is only used in PRSS sign.
+        /// </summary>
+        public System.Security.Cryptography.CngKey PrivateKey { get; set; }
```

Comment:
`SignPackageRequest` should implement `IDisposable` and dispose of this member and `Certificate`.  Instances of `SignPackageRequest` should be wrapped in a `using` statement.